README.md

```diff
@@ -82,7 +82,7 @@ pak::pkg_install("cmu-delphi/epidatr@dev")
 ```
 
 Our CRAN listing is
-[here](https://CRAN.R-project.org/package=epidatr/index.html).
+[here](https://cran.r-project.org/web/packages/epidatr/index.html).
```

Comment:
I think the original is closer to correct because it's considered canonical by CRAN (I don't fully understand _why_ they want things that way, but [this is an example](https://stackoverflow.com/questions/64257455/cran-url-not-in-canonical-form) of the issue CRAN would raise). So, if anything the URL should be 
```suggestion
[here](https://CRAN.R-project.org/package=epidatr).
```